ecosystem/indexer-grpc/indexer-grpc-table-info/src/indexer-async-v2/src/lib.rs

```diff
@@ -0,0 +1,344 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod db;
```

Comment:
Can you make a note that this is a copy of file X and that at the end of the migration the other file will be deleted.